src/schema/api.ts

```diff
@@ -132,6 +133,10 @@ export class API {
       );
     }
     this.mainServiceName = options.mainServiceName || serviceNamesList[0];
+    // For generating keywords in package.json
+    this.uniqkeywords = [
```

Comment:
nit: this should be `uniqKeywords`, we use camel case for variables everywhere

Reply:
👍